.github/workflows/ci.yml

```
@@ -7,26 +9,42 @@ on:
     branches: ["main"]
 
 jobs:
-  lint:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v4
-      - uses: ./.github/actions/setup
+          version: '10.18.2'
```

actionlint (1.7.8)

could not parse as YAML: did not find expected key

(syntax-check)